src/components/markdown-htmlview.component.js

```diff
@@ -0,0 +1,335 @@
+import React, { Component } from 'react';
+import { Dimensions, StyleSheet, View, Text, Platform } from 'react-native';
+import HTMLView from 'react-native-htmlview';
+import marked from 'marked';
+import { TableWrapper, Table, Cell } from 'react-native-table-component';
+import SyntaxHighlighter from 'react-native-syntax-highlighter';
+import { github as GithubStyle } from 'react-syntax-highlighter/dist/styles';
+
+import { ImageZoom } from 'components';
+import { colors, emojis, fonts, normalize } from 'config';
+
+const lightFont = {
+  ...fonts.fontPrimaryLight,
+};
+
+const regularFont = {
+  ...fonts.fontPrimary,
+};
+
+const textStyleLight = {
+  ...lightFont,
+  fontSize: Platform.OS === 'ios' ? normalize(11) : normalize(12),
+  color: colors.primaryDark,
+};
+
+const textStyleRegular = {
+  ...regularFont,
+  fontSize: Platform.OS === 'ios' ? normalize(11) : normalize(12),
+  color: colors.primaryDark,
+};
+
+const textStyle = Platform.OS === 'ios' ? textStyleLight : textStyleRegular;
+
+const boldStyle = {
+  ...textStyle,
+  ...fonts.fontPrimarySemiBold,
+};
+const italicStyle = { ...textStyle, ...fonts.fontPrimaryItalic };
+const hrStyle = { borderBottomWidth: 2, borderBottomColor: colors.greyDark };
+const underlineStyle = { textDecorationLine: 'underline' };
+const codeStyle = {
+  ...fonts.fontCode,
+  backgroundColor: colors.greyMidLight,
+  fontSize: normalize(9),
+};
+const linkStyle = { ...boldStyle, color: colors.primaryDark };
+
+const styles = {
+  b: boldStyle,
+  strong: boldStyle,
+  i: italicStyle,
+  em: italicStyle,
+  u: underlineStyle,
+  pre: { ...codeStyle, padding: 5, marginBottom: 10 },
+  code: codeStyle,
+  span: textStyle,
+  p: { ...textStyle, margin: 0, padding: 0 },
+  h1: {
+    ...fonts.fontPrimarySemiBold,
+    ...hrStyle,
+    fontSize: normalize(24),
+  },
+  h2: {
+    ...fonts.fontPrimarySemiBold,
+    ...hrStyle,
+    fontSize: normalize(20),
+  },
+  h3: { ...fonts.fontPrimarySemiBold, fontSize: normalize(18) },
+  h4: { ...fonts.fontPrimarySemiBold, fontSize: normalize(16) },
+  h5: { ...fonts.fontPrimarySemiBold, fontSize: normalize(14) },
+  h6: { ...fonts.fontPrimarySemiBold, fontSize: normalize(12) },
+  hr: { ...hrStyle },
+  li: textStyle,
+  a: linkStyle,
+};
+
+const stylessheet = StyleSheet.create(styles);
```

Comment:
Another minor nitpick :eyes:, but could we keep the variable camelcase like so: `styleSheet`. Also think it might be nicer to name it `styleSheet` rather than `stylesSheet` since it matches the RN API